cli/ingress-controller/main.go

```diff
@@ -392,14 +404,40 @@ func main() {
 				Client: kongClient,
 			},
 		}
+		var cert tls.Certificate
+		if cliConfig.AdmissionWebhookCertPath != "" && cliConfig.AdmissionWebhookCert != defaultAdmissionWebhookCertPath {
```

Comment:
It seems that the comparison here are wrong. Should this not be checking if AdmissionWebhookCertPath is not equal to default value and not empty AdmissionWebhookCert?

Same applies for key.

Reply:
Similar variable names :woman_facepalming: